# Wet Lab
## Experimental Workflow and Goals

The wet lab team’s fundamental purpose for this project is to test and validate
generated RBD (receptor-binding domain) sequences generated by the dry lab team
_in vitro_. While computational models can predict RBD–ligand interactions,
_in vitro_ experiments are required to confirm functional activity and ensure
that predicted efficacy translates into measurable biological and therapeutic
outcomes.

To assess the _in vitro_ functionality and efficacy of candidate RBDs, we
leverage the fact that the sequences coding for them are, in general, highly
modular for phages, allowing them to be interchanged with each other without
disrupting the entire viral particle. Thus, we endeavour to swap the wild-type
(WT) RBD sequence within a given phage with our generated one, which enables
us to test the latter’s ability to bind to bacterial targets with specific
receptors of interest. This will be done through a combination of homologous
recombination and CRISPR-Cas counterselection.

To achieve this, two plasmids are co-transformed into a bacterium containing
a prophage (an _E. coli_ K12 lysogen). The first plasmid contains the generated
RBD with homology arms flanking the upstream and downstream regions of the
prophage RBD – this will undergo recombination to swap the generated RBD into
the prophage, replacing the WT counterpart. This process has very low efficiency,
which necessitates the use of a second plasmid containing Cas machinery and an
sgRNA that targets the WT RBD. The sgRNA complexes with the Cas enzymes to cut
prophages containing the WT RBD sequence selectively, which has a twofold benefit
of counterselecting against non-recombined prophages and creating double-stranded
breaks that facilitate the RBD recombination/swapping process. Following this,
the swapped phages are rebooted, and efficacy is then evaluated by phage spotting
assays, where successfully engineered phages are expected to bind glycans or
proteins that WT variants cannot recognize.

The workflow described in the following pages is divided into two main sections:
plasmid preparation and phage preparation. The plasmid preparation stage
describes the construction and assembly of the sgRNA- and RBD-containing plasmids.
Once sequencing confirms that both have been assembled correctly, they are then
co-transformed into K12 lysogen cells, which transition over to the phage
preparation stage. Swapping and phage rebooting takes place within K12, where
phages are extracted and their RBD specificity is assessed through phage spotting.

<Figure
  src={"https://static.igem.wiki/teams/5708/wetlab/wet-lab-wiki-overview.webp"}
  alt="Figure 1: Experimental workflow overview"
  caption="Figure 1: Experimental workflow overview" />
