---
---
import Figure from '../components/Figure.astro';

# Project Description

## Motivation
### The rise of antibiotic resistance and phage therapy

Antibiotics have revolutionized healthcare, preventing millions of deaths from bacterial infections since their discovery in 1928 @Adedeji2016TreasureAntibiotics. However, misuse and overuse of these drugs in medicine and the food industry has promoted the development of **antibiotic resistance** (ABR) – the ability of microbes, particularly bacteria, to withstand antibiotic activity @Strathdee2023PhageTherapyCell. This issue, when combined with broader resistance from other microbes, represents one of humanity’s most pressing global health challenges. In 2021, 4.71 million deaths were associated with drug-resistant bacterial infections, and between now and 2050, the cumulative death toll could exceed 39 million @Naghavi2024GlobalAMRForecasts @Kozlov2024NatureNewsAMR39M. Frequently mentioned in the fight against ABR are the ESKAPE pathogens @Santaniello2020ESKAPEDogs and the World Health Organization’s priority pathogen list @WHO2024BPPL, with resistant variants of _Acinetobacter baumannii_, _Enterobacterales_, and _Mycobacterium tuberculosis_ at the forefront.

The rising urgency of ABR has sparked renewed interest in **phage therapy** – a promising alternative treatment that employs viruses called phages to target and kill resistant infections. Phages are the naturally occurring predators of bacteria and the most abundant biological entity in the biosphere @Strathdee2023PhageTherapyCell. They have co-evolved with bacteria over billions of years, granting them sophisticated attack systems and replication strategies to clear and control bacterial populations @Murtazalieva2024PhageAntiDefence. So far, phage therapy has had success in a number of high-profile cases where antibiotic treatment has failed @CNN2022PhageStory @CBC2024PhageCanada.

### Making phage therapy faster with Mystiphage

Given the immense diversity of phages, only a subset with desirable characteristics (i.e. lytic, appropriate host range) are amenable for therapy @PMC6469166PhageTherapyReview. Host range in particular is crucially important, and is primarily determined by phages’ receptor-binding proteins (RBPs), which target complementary surface receptors on bacterial membranes @Pharmaceuticals2023PhagesHostRange. If the RBPs and receptor don’t match, the phage will have incorrect host specificity and will be unable to bind to, infect, and kill the target bacteria – rendering it unsuitable for therapy @Pharmaceuticals2023PhagesHostRange @Gelman2021ClinicalPhageMicrobiology.

The process of finding phages with the correct RBPs/host range for the desired pathogen is known as **phage matching** @Gelman2021ClinicalPhageMicrobiology, which unfortunately is extremely laborious and time-consuming @Frontiers2025FCIMB1611857. Needing to find the ‘right’ phage for an infection contributes to **single-digit fulfillment rates** @Pirnay2024BT100NatureMicrobiology and **months-long wait times** @OFID2020CompassionateUsePhage for many phage therapy cases – much too long for critically ill patients. Without a faster matching process, phage therapy will not scale to the millions of possible future infections where it might be needed when antibiotics fail.

To address this problem, we are developing **Mystiphage**, a combined generative AI and hardware platform to (1) reduce the phage matching process from months to **hours**, and (2) deliver our AI-modified phages to critical target sites to stop the spread of ABR.

## PHORAGER: Phage Host-Optimized Receptor-Activated Generative Engineering Repository
Generative protein models represent a profound advancement in protein engineering, enabling the creation of diverse protein binders for various targets @Hsu2024GenerativeModelsProtein @Cao2022DesignProteinBinders. However, three main issues prevent these models from effectively generating specific RBPs:

1.  Bacterial receptors are incredibly diverse. Since each RBP is highly specific to each given receptor, generative models would need to encompass a vast range of sequence and structural diversity to generalize across necessary receptor classes. This makes de novo generation very difficult.
2.  Many bacterial receptors, such as flexible glycans, are extremely challenging to represent computationally, and current generative models struggle to design binders for them.
3.  Many receptors lack solved or annotated RBP-receptor complexes, leaving generative models with little data to train on or fine-tune.

To overcome these challenges, we built PHORAGER – a generative phage bank to develop RBPs for a diverse set of targets. First, PHORAGER takes in a bacterial genome as input and identifies its receptor types. From this, it finds bacteria that are most likely to have these receptors and the corresponding phages that infect them. If these phages have high binding affinity to the input, they are recommended to the user; if not, they are used as a template for a generative pipeline.

Here, the receptor binding domains (RBDs) from the highest affinity phages are input into ESM3 – a masked protein language model, to hallucinate new residues that confer greater affinity to the target receptor. This is tested by co-folding the outputs and the target using Boltz-2 – a diffusion-based protein folding model, and observing the resulting structure and metrics. By repeating this process over many trials using Markov Chain Monte Carlo (MCMC), PHORAGER converges on better and better binders with each iteration. For silico validation, molecular docking with Haddock 3 is used. Promising candidates are then codon-optimized and sent to the wet lab team for in vitro validation.
  

## PHORAGER’s outputs are validated in the wet lab

To validate the RBDs generated from PHORAGER, the wet lab team seeks to perform a phage swapping workflow. Here, the generated RBDs are cloned into plasmids via Gibson Assembly and transformed into a lysogenic E. coli K12 strain. Once inside, the generated RBD undergoes homologous recombination with the K12 prophage to trade places with its wild type RBD. At the same time, an sgRNA sequence targeting the wild type RBD is cloned to a CRISPR-Cas plasmid and transformed into the same K12 strain, enabling counterselection against non-recombined prophages.

Afterwards, the prophage is induced to enter the lytic cycle via the antibiotic mitomycin C, allowing the mutant phages to be collected and purified. To test their ability to infect bacteria, the mutants are spotted on lawns of K12 with diverse receptor types. The results from this spotting experiment will then be sent back to the dry lab team to further inform the PHORAGER pipeline.

 
## EVADE: Electroceutical Vehicle for Antimicrobial Delivery

While AI-powered phage therapies hold great promise for treating infections, its applications can extend to other target sites. In the GI tracts of both humans and livestock exists the gut resistome – the collection of antibiotic resistance genes (ARGs) directly associated with the development and spread of ABR infections @SciDirect2025S2772416625000464. The resistome is dynamic in time and space, and can become enriched for harmful ARGs when placed under selective pressure from antibiotics @SciDirect2025S2772416625000464. This is especially a concern in agriculture, which accounts for over half of all antibiotic use (in much less regulated fashions compared to clinical settings) @terKuile2016LowAntibioticsAgriculture.

Thus, we built EVADE – an electroceutical, bacteria-sensing pill that can deliver AI-modified phage therapies directly to bacterial infection sites in the gut. EVADE is designed to traverse the GI tracts of humans and livestock and detect biomarkers of bacterial growth, releasing phage therapies and possible adjuvants at high concentrations to kill resistant bacteria.

We are exploring how EVADE can be used as a generalized phage delivery device to the GI tracts of both humans and animals to directly target sites of ARG enrichment. By conditionally deploying phage therapies to optimize the gut resistome, we can stop the spread of ABR by striking a critical reservoir of resistance genes.

## Human practices lays explores the ethics of Mystiphage

At iGEM Toronto, human practices represent a commitment to ethically grounded and socially responsible innovation. For Mystiphage, this meant conducting over 60 stakeholder interviews spanning academia, industry, law and public health. This included professors, intellectual property lawyers, and even other biotech founders, all of whom helped inform our workflows across our wet lab, dry lab, hardware, and entrepreneurship subteams. This integrative approach ensured that expert insights were not only heard but also actively embedded into our design and decision-making processes. In parallel, we also developed a series of educational and outreach materials to raise public awareness about phage therapy and the global challenge of antibiotic resistance. These included a sustainability paper, regulatory framework analysis, socioeconomic white paper, newsletter infographics, and targeted social media campaigns. Ultimately, our human practices aims to ensure that our project is inclusive, transparent, and representative, both in its conceptual foundation and its real-world application.

## Commercializing Mystiphage
This year, we sought to develop Mystiphage into a go-to-market stage company, integrating all aspects of our project into investor-ready deliverables. Through an incubator program, we refined our problem statement, value proposition, and business model. We also developed a fully-fledged business plan, 6-minute pitch, and other items necessary to engage in pre-seed and seed funding rounds.

By commercializing Mystiphage, we intend to navigate the complex regulatory landscape of phage therapy and generative protein therapies to translate our work to a global community in need.